lib/core/datatable/components/datatable/datatable.component.ts

```diff
@@ -474,6 +474,12 @@ export class DataTableComponent implements AfterContentInit, OnChanges, DoCheck,
         this.clickObserver.next(dataRowEvent);
     }
 
+    onRowEnterKeyDown(row: DataRow, e: KeyboardEvent) {
+        if (e.key === 'Enter') {
+            this.onKeyboardNavigate(row, e);
+        }
+    }
+
     onRowKeyUp(row: DataRow, e: KeyboardEvent) {
```

Comment:
avoid use os single char name variable prefer event over e that help the redability of the code when the variable is used far from the type